### ISSUE TITLE
Shady1 random encounter wrong raider script

### ISSUE THREAD

**Issue by Thrax-x** [rotators/Fo1in2#173]
In RNDDESRT.ssl script, procedure Shady1, line 829 - Critter_script := SCRIPT_GENRAIDA;
Correct values shoud be Critter_script := SCRIPT_GENRAIDB;
After all raiders are dead and you've killed at least one of them, the victims should give you reward, but raiders are not counted as killed with the wrong script name.

There is also a minor bug in GVAR 1289 name definition in global.h.
It's defined as GVAR_WATER_THIEF_2 (1289), but it has nothing to do with Vault 13 Water thief. It's the counter of remaining raiders alive.
It's used in random encounter scripts only:
dealer.ssl
duc.ssl
gengrdc.ssl
genraidb.ssl
peasvict.ssl
rndcoast.ssl
rnddesrt.ssl

**Comment by Lexx2k** [comment]
1. Not a bug. Not every critter / encounter has to give you a reward at the end of the fight.
2. That's just old leftover stuff from the original game. I'll rewrite it some day.

**Comment by Thrax-x** [comment]
1. Maybe you're right. I've checked the wiki and there is no information about the reward of this encounter unlike the Coast4 encounter. But the wiki describes the original game with all the bugs, so this can be a vanilla bug.
2. It has no inpact to gameplay. I've checked files of original fallout a Fixt mod and this came from Fixt. Author used Gvars 289-292 (1289-1292 in Et tu) for a new purpose and forget to change the name of this one.
